pkg/volume/util/operationexecutor/operation_executor.go

```diff
@@ -211,6 +211,9 @@ func generateVolumeMsg(prefixMsg, suffixMsg, volumeName, details string) (simple
 
 // VolumeToAttach represents a volume that should be attached to a node.
 type VolumeToAttach struct {
+	// MultiAttachErrorReported shows whether the multi attach err is reported or not
```

Comment:
How about `MultiAttachErrorReported indicates whether the multi-attach error has been reported for the given volume. It is used to to prevent reporting the error from being reported more than once for a given volume.`